### ISSUE TITLE
Can't connect to (almost) any server in my country.

### ISSUE THREAD

**Issue by holaN1** [haxball/haxball-issues#141]
I can't connect to any server in my country, except a few. Not everyone can connect to my server, those who can connect to my server, I can connect to them too. However, I can connect to all servers in other countries.
I disabled adblockers, I tried other browsers in vain. I don't know what to do.
![screenshot-2017-11-7 haxball html5 alpha 1](https://user-images.githubusercontent.com/33462694/32512833-f927ec30-c400-11e7-89c1-0283bd4c4578.png)
![screenshot-2017-11-7 haxball html5 alpha](https://user-images.githubusercontent.com/33462694/32512838-fd5b265a-c400-11e7-8a30-3a17d0178d65.png)

**Comment by loveuhaxball** [comment]
http://html5.haxball.com/reverse_setting.htmsl

select enable and try again

**Comment by r2308145** [comment]
I think since the last update the game should automatically set itself for the reverse connection or vice versa

**Comment by basro** [comment]
That is correct, HaxBall will now try reverse connection if connecting normally fails.

If two players have the same issue where only reverse connection works, then they wont be able to connect to each other since it's always going to be "forward" for one of them. I believe this is probably what is happening for you @holaN1.

**Comment by Blaze52** [comment]
Getting similar issues, although I don't get the "Disconnected – Failed" message, I get a slightly different message.

![screen shot 2017-11-09 at 5 01 48 pm](https://user-images.githubusercontent.com/33357405/32631955-d55c8e8a-c56f-11e7-8d3e-9efb29725abe.jpg)

<img width="512" alt="image" src="https://user-images.githubusercontent.com/33357405/32631968-df778ac8-c56f-11e7-9cb4-b5ea51289533.png">

**Comment by basro** [comment]
@Blaze52, do you get this every time or only sometimes?

**Comment by Blaze52** [comment]
@basro It happens every single time, I can't connect to any html5 room. I haven't tried hosting yet, but I assume people wouldn't be able to join my room either.

**Comment by Nenegol** [comment]
same problem...cant connect any server in html @basro
